ui/pages/confirmations/confirmation/confirmation.js

```diff
@@ -195,6 +195,30 @@ function useTemplateState(pendingConfirmation) {
   return [templateState];
 }
 
+function Header({ confirmation, isSnapCustomUIDialog, onCancel }) {
+  const { count } = useConfirmationNavigation();
+  const { origin } = confirmation ?? {};
+
+  const hideSnapBranding = useSelector((state) =>
+    getHideSnapBranding(state, origin),
+  );
+
+  const requiresSnapHeader = isSnapCustomUIDialog && !hideSnapBranding;
+
+  if (count <= 1 && !requiresSnapHeader) {
+    return null;
+  }
+
+  return (
+    <Box style={{ width: '100%' }}>
+      <Nav confirmationId={confirmation?.id} />
+      {requiresSnapHeader && (
+        <SnapAuthorshipHeader snapId={origin} onCancel={onCancel} />
```

Comment:
Looks like nesting our header breaks the box shadow on it 🤔

Reply:
Fixed.

Reply:
This still seems broken to me, I don't think we would want the box shadow to apply "between" the nav and the authorship header.

Reply:
Done, box shadow only on the bottom.